Godeps/_workspace/src/k8s.io/kubernetes/plugin/pkg/admission/namespace/lifecycle/admission.go

```diff
@@ -74,6 +75,11 @@ func (l *lifecycle) Admit(a admission.Attributes) (err error) {
 		return nil
 	}
 
+	// always allow access review checks.  Returning status about the namespace would be leaking information
+	if isAccessReview(a) {
```

Comment:
sgtm

On Fri, Jul 8, 2016 at 1:41 PM, David Eads <notifications@github.com> wrote:

> In
> Godeps/_workspace/src/k8s.io/kubernetes/plugin/pkg/admission/namespace/lifecycle/admission.go
> <https://github.com/openshift/origin/pull/9740#discussion_r70111848>:
>
> > @@ -74,6 +75,11 @@ func (l *lifecycle) Admit(a admission.Attributes) (err error) {
> >  		return nil
> >  	}
> >
> > +	// always allow access review checks.  Returning status about the namespace would be leaking information
> > +	if isAccessReview(a) {
>
> and just make this a check on isIgnoredResource when we look to push this
> upstream.
>
> Map check seems to make more sense. Mess it when we upstream this?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9740/files/eb30525dab33068b8d23596787f408163fcb101b..343a20032f177ff10776c8887e591e6e3f17cb92#r70111848>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AF8dbKJRKZHhGXefcQqoOEcZNku5yN0dks5qTou5gaJpZM4JHalQ>
> .
>